aws/lambdas/readEventPage/index.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const http      = require('http');
+const cheerio   = require('cheerio');
+
+let $;
+
+const requestOptions = {
+    host: 'www.uzletiliga.hu',
+    path: '/eredmenyek/matches.php'
+};
+
+const addQueryParams = (queryParams, requestOptions) => {
+    requestOptions.path += Object.keys(queryParams).reduce((queryParamsString, paramKey) => {
+        return queryParamsString += `${paramKey}=${queryParams[paramKey]}`;
```

Comment:
I believe an `&` is required somewhere here if there is more than one query param